app/services/send_applications_to_provider.rb

```diff
@@ -0,0 +1,7 @@
+class SendApplicationsToProvider
```

Comment:
Can you add a comment here explaining that it's a clockwork job that calls this class periodically?